manala.graylog_sidecar/tasks/install.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: install > Deb
+  apt:
+    deb: "{{ item }}"
+  with_items: "{{ lookup(
+      'manala_graylog_sidecar_deb',
```

Comment:
Why this lookup ? We handle only *.deb in our install tasks. 

cc @nervo

Reply:
revert to not using lookup